### PR TITLE
Feature: xcframework

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

release this framework in XCFramework format

## Dependencies

Needs releases with:

- [x] https://github.com/wireapp/wire-ios-shared-resources/pull/50
- [x] https://github.com/wireapp/wire-ios-system/pull/59

### CODE REVIEW

File: WireTesting.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -527,6 +517,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 09F1863F1B6903A4007A3DA6 /* WireTesting.xcconfig */;
 			buildSettings = {
+				DEVELOPMENT_TEAM = "";
```
Comment (billypchan) [inline WireTesting.xcodeproj/project.pbxproj:520]:
```suggestion
```

File: WireTesting.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -516,6 +507,7 @@
...
```
Comment (billypchan) [inline WireTesting.xcodeproj/project.pbxproj:510]:
```suggestion
```

File: Tests/WireTestingTests.m [hunk 3]
```diff
@@ -1,6 +1,6 @@
 // 
 // Wire
-// Copyright (C) 2016 Wire Swiss GmbH
+// Copyright (C) 2021 Wire Swiss GmbH
```
Comment (billypchan) [inline Tests/WireTestingTests.m:3]:
```suggestion
// Copyright (C) 2016 Wire Swiss GmbH
```

File: Source/Public/NSUUID+WireTesting.h [hunk 4]
```diff
@@ -1,6 +1,6 @@
 // 
 // Wire
-// Copyright (C) 2016 Wire Swiss GmbH
+// Copyright (C) 2021 Wire Swiss GmbH
```
Comment (billypchan) [inline Source/Public/NSUUID+WireTesting.h:3]:
```suggestion
// Copyright (C) 2016 Wire Swiss GmbH
```

File: Source/Public/NSOperationQueue+WireTesting.m [hunk 5]
```diff
@@ -1,6 +1,6 @@
 // 
 // Wire
-// Copyright (C) 2016 Wire Swiss GmbH
+// Copyright (C) 2021 Wire Swiss GmbH
```
Comment (billypchan) [inline Source/Public/NSOperationQueue+WireTesting.m:3]:
```suggestion
// Copyright (C) 2016 Wire Swiss GmbH
```

File: Source/Public/NSData+WireTesting.h [hunk 6]
```diff
@@ -1,6 +1,6 @@
 // 
 // Wire
-// Copyright (C) 2016 Wire Swiss GmbH
+// Copyright (C) 2021 Wire Swiss GmbH
```
Comment (billypchan) [inline Source/Public/NSData+WireTesting.h:3]:
```suggestion
// Copyright (C) 2016 Wire Swiss GmbH
```

File: Source/Public/FakeGroupContext.swift [hunk 7]
```diff
@@ -1,6 +1,6 @@
 // 
 // Wire
-// Copyright (C) 2016 Wire Swiss GmbH
+// Copyright (C) 2021 Wire Swiss GmbH
```
Comment (billypchan) [inline Source/Public/FakeGroupContext.swift:3]:
```suggestion
// Copyright (C) 2016 Wire Swiss GmbH
```

File: Source/Public/NSData+WireTesting.m [hunk 8]
```diff
@@ -1,6 +1,6 @@
 // 
 // Wire
-// Copyright (C) 2016 Wire Swiss GmbH
+// Copyright (C) 2021 Wire Swiss GmbH
```
Comment (billypchan) [inline Source/Public/NSData+WireTesting.m:3]:
```suggestion
// Copyright (C) 2016 Wire Swiss GmbH
```

**Review by typfel — APPROVED** [review]

**Review by David-Henner — APPROVED** [review]

File: azure-pipelines.yml [hunk 9]
```diff
@@ -3,7 +3,7 @@ resources:
   - repository: wire-ios-shared-resources
     type: github
     name: wireapp/wire-ios-shared-resources
-    ref: refs/heads/master # Branch to fetch the jobs template from
+    ref: refs/heads/feature/xcframework # Branch to fetch the jobs template from
```
Comment (billypchan) [inline azure-pipelines.yml:6]:
```suggestion
    ref: refs/heads/master # Branch to fetch the jobs template from
```

**Comment by billypchan** [comment]
@zenkins release major

**Pull request merged**